otter/test/convergence/test_service.py

```diff
@@ -817,7 +823,8 @@ def setUp(self):
                    links=freeze([{'href': 'link2', 'rel': 'self'}]))
         )
         self.state_active = {}
-        self.cache = [thaw(self.servers[0].json), thaw(self.servers[1].json)]
+        self.cache = [thaw(self.servers[0].json.set('_is_as_active', True)),
+                      thaw(self.servers[1].json.set('_is_as_active', True))]
```

Comment:
Well, this comes down to how the tests should be updated. I had a choice between
- Changing `self.cache` and the lb_nodes returned by the fake gathering globally, and then removing the lb_nodes returned in certain test cases.
- Changing lb_nodes returned by gathering in the tests cases where convergence succeeds, which I think would require changing `self.cache` in those test cases.
- Some other combination of fiddling with tests.

I went with the first option. We can discuss this because I'm not going to pretend I know the best way to update the tests for this change.